extensions/interactions/base_test.py

```diff
@@ -553,21 +563,24 @@ def test_default_interactions_are_valid(self):
 
             # Check that the html template includes js script for the
             # interaction.
-            self.assertTrue(
-                'oppia-interactive-%s.component.ts' %
-                hyphenated_interaction_id in ts_file_content or (
-                    'oppia-interactive-%s.directive.ts' %
-                    hyphenated_interaction_id in ts_file_content))
-            self.assertTrue(
-                'oppia-response-%s.component.ts' %
-                hyphenated_interaction_id in ts_file_content or (
-                    'oppia-response-%s.directive.ts' %
-                    hyphenated_interaction_id in ts_file_content))
-            self.assertTrue(
-                'oppia-short-response-%s.component.ts' %
-                hyphenated_interaction_id in ts_file_content or (
-                    'oppia-short-response-%s.directive.ts' %
-                    hyphenated_interaction_id in ts_file_content))
+            # TODO(#10212): Remove this as soon as the learner's flow is added
```

Comment:
#10212 doesn’t seem like the right reference here. Please fix.

Reply:
#10460 updated.